src/Jira/Api.php

```diff
@@ -173,14 +173,14 @@ protected function clearLocalCaches()
 	/**
 	 * Get fields definitions.
 	 *
-	 * @return array
+	 * @return array|false
```

Comment:
Luckily changed methods aren't 100% of public class interface. Please create some basic tests for them (at least confirm, that they return array all the time).